src/mpi/romio/adio/ad_gpfs/ad_gpfs_open.c

```diff
@@ -12,6 +12,8 @@
  *   See COPYRIGHT notice in top-level directory.
  */
 
+#define _GNU_SOURCE
+
 #include "ad_gpfs.h"
```

Comment:
a bare _GNU_SOURCE define seems wrong.   We probably need AC_USE_SYTEM_EXTENSIONS but we don't want to turn that on 100% of the time either (we don't want to find ourselves relying on extensions that turn out not to be present on some platform of interest). 

I don't know how many FreeBSD GPFS clients running ROMIO we have out there in the world -- my guess is none -- but I don't think FreeBSD has stat64 .   Since we are looking for blocksize, not inodes, uid, or gid,  plain stat should suffice.   I cannot find our original justification for using stat64 .